src/current/cockroachcloud/private-clusters.md

```diff
@@ -28,7 +28,7 @@ Private clusters are not available for [CockroachDB {{ site.data.products.advanc
 ## Create a private cluster
 
 On GCP, new CockroachDB {{ site.data.products.advanced }} clusters are private by default.
-On AWS, newly CockroachDB {{ site.data.products.advanced }} with enhanced security features clusters deployed on AWS are private by default.
+On AWS, newly CockroachDB {{ site.data.products.advanced }} with advanced security features clusters deployed on AWS are private by default.
```

Comment:
@jhlodin All Azure cluster are private by default. But one thing to note is that we don't support PCI-DSS on Azure yet

Reply:
I want to hold off on making more edits to this file specifically, because there is another separate discussion I want to have over use of the seemingly outdated term "private cluster" and whether this doc should continue to exist as-is. See DOC-14194 for more context.

So I'd prefer to leave this untouched for now to avoid scope creep on this PR.